executor.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal EnableDelayedExpansion
+
+if not defined VisualStudioVersion (
+    if defined VS140COMNTOOLS (
+        call "%VS140COMNTOOLS%\VsDevCmd.bat"
+        goto :Run
+    )
+
+    if defined VS120COMNTOOLS (
+        call "%VS120COMNTOOLS%\VsDevCmd.bat"
+        goto :Run
+    )
+
+    echo Error: Visual Studio 2013 or 2015 required.
+    echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/developer-guide.md for build instructions.
```

Comment:
That link is broken, it should point to https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md now.